### ISSUE TITLE
[Accessibility: ACC-255.1] Button: No meaningful text alternative for non-text UI.

### ISSUE THREAD

**Issue by divyamalhotra488** [SAP/fundamental-ngx#12613]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
For all non-text UI elements provided text alternatives should be meaningful and fulfill the purpose of the UI.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Component used: [Flexible Column Layout](https://sap.github.io/fundamental-ngx/#/core/flexible-column-layout)
For the ‘Slider’ within the screen, the text alternative is provided as 'Undefined', which doesn’t define the meaning of the UI.
<img width="452" alt="image" src="https://github.com/user-attachments/assets/4f5163e5-624c-4c7a-87d0-cb3f147f14e8">

#### Please provide stackblitz example(s).
You can try ngx examples, issue is reproducible there.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes, issue is raised by an accessibility expert.

#### Did you check the documentation and the API?
Yes

#### Did you search for similar issues?
Yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
@divyamalhotra488 you need to open this ticket to the application responsible for that UI. They need use the API and pass those titles.
